common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbUtils.java

```diff
@@ -647,6 +897,15 @@ public static String decodeURI(String uri) {
         return URLDecoder.decode(uri, StandardCharsets.UTF_8);
     }
 
+    public static void newError(String message) {
+        newError(message, null);
+    }
+
+    public static void newError(String message, Object value) {
+        String msg = value == null ? message : message + " Value = " + value;
```

Comment:
Consider renaming the value part to " for value X"

Reply:
ok